pset-iterators.js

```diff
@@ -66,6 +68,11 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+const presidentsNamedJames = presidents.filter(function(president) {
+  return president.president.startsWith("James")
+});
+
+console.log(presidentsNamedJames)
```

Comment:
@jl1982 Good Job here using the string method `startsWith()` inside your `filter` method to return all of the presidents whose name starts with 'James'.

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L62)